lib/bolt/outputter/human.rb

```diff
@@ -102,7 +102,8 @@ def print_task_info(task)
 
         task['parameters'].each do |k, v|
           pretty_params << "- #{k}: #{v['type']}"
-          pretty_params << "\n    #{v['description']}\n" if v['description']
+          pretty_params << "\n    #{v['description']}" if v['description']
+          pretty_params << "\n"
```

Comment:
maybe just give each line it's own \n ie
"- #{k}: #{v['type']}\n"
"    #{v['description']}\n"